roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -27,8 +27,9 @@ debug = {{ nova.logging.debug }}
 # Scheduler #
 scheduler_default_filters = {{ nova.scheduler_default_filters }}
 
-default_availability_zone = {{ nova.default_availability_zone }}
+{% if nova.default_availability_zone is defined -%}
```

Comment:
default_availability_zone is what zone a compute node will consider itself in, unless otherwise specified.

default_schedule_zone is what zone the scheduler will pick if one isn't provided by the client.

We'll leave the ability to set a default schedule zone, but there wasn't really a reason to keep the default availability zone setting for compute nodes.